modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/PreferenceAPIIntegrationUITestCase.java

```diff
@@ -65,30 +69,27 @@ public class PreferenceAPIIntegrationUITestCase extends OAuth2ServiceAbstractInt
 
     private IdentityProvider superTenantResidentIDP;
     private IdentityProviderMgtServiceClient superTenantIDPMgtClient;
+    private ServerConfigurationManager serverConfigurationManager;
     private String recoveryEndpoint;
-    private final String activeTenant;
+    private String activeTenant;
     private final String OIDC_APP_NAME = "playground23";
     private String oidcAppClientId = "";
 
-    @DataProvider(name = "configProvider")
-    public static Object[][] configProvider() {
-
-        return new Object[][]{{TestUserMode.SUPER_TENANT_ADMIN}, {TestUserMode.TENANT_ADMIN}};
-    }
-
-    @Factory(dataProvider = "configProvider")
-    public PreferenceAPIIntegrationUITestCase(TestUserMode userMode) throws Exception {
-
-        super.init(userMode);
-        AutomationContext context = new AutomationContext("IDENTITY", userMode);
-        this.activeTenant = context.getContextTenant().getDomain();
-
-    }
-
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
 
         super.init();
+        String carbonHome = Utils.getResidentCarbonHome();
+        File defaultConfigFile = getDeploymentTomlFile(carbonHome);
+        File challengeQuestionsConfigFile = new File(
+                getISResourceLocation() + File.separator + "challenge-questions" + File.separator + ADD_CHALLENGE_QUESTIONS_CONFIG);
+        serverConfigurationManager = new ServerConfigurationManager(isServer);
+        serverConfigurationManager.applyConfigurationWithoutRestart(challengeQuestionsConfigFile, defaultConfigFile, true);
+        serverConfigurationManager.restartGracefully();
+
+        super.init();
+        AutomationContext context = new AutomationContext("IDENTITY", TestUserMode.SUPER_TENANT_ADMIN);
```

Comment:
Previously this test was executed for super tenant and a tenant. Why do we specifically need to avoid the tenant execution?